pkg/fanal/handler/misconf/misconf.go

```diff
@@ -39,6 +39,7 @@ func init() {
 const version = 1
 
 type misconfPostHandler struct {
+	options  artifact.Option
```

Comment:
What if we only store `options.MisconfScannerOption.FilePatterns` here?
I think we don't need to keep all "options".

Reply:
Yeah, that's true, done :+1: